group_wikis_test.go

```diff
@@ -35,15 +35,15 @@ func TestGetGroupWikiPage(t *testing.T) {
 	mux.HandleFunc("/api/v4/groups/1/wikis/deploy",
 		func(w http.ResponseWriter, r *http.Request) {
 			testMethod(t, r, http.MethodGet)
-			fmt.Fprint(w, `{"content":"content here","format":"asciidoc","slug":"deploy","title":"deploy title"}`)
+			fmt.Fprint(w, `{"content":"content here","format":"asciidoc","slug":"deploy","title":"deploy title","encoding":"UTF-8"}`)
 		})
 
-	groupwiki, _, err := client.GroupWikis.GetGroupWikiPage(1, "deploy")
+	groupwiki, _, err := client.GroupWikis.GetGroupWikiPage(1, "deploy", &GetGroupWikiPageOptions{})
 	if err != nil {
 		t.Errorf("GroupWikis.GetGroupWikiPage returned error: %v", err)
 	}
 
-	want := &GroupWiki{Content: "content here", Format: WikiFormatASCIIDoc, Slug: "deploy", Title: "deploy title"}
+	want := &GroupWiki{Content: "content here", Encoding: "UTF-8", Format: WikiFormatASCIIDoc, Slug: "deploy", Title: "deploy title"}
```

Comment:
Maybe nice to format these test bodies similar to how they are formatted in `wikis_test.go`?

Reply:
I went through and reformatted the `want`s and also reformatted the JSON strings in both files a little bit. Let me know if that was what you were looking for!